### ISSUE TITLE
lintspaces is run on binary files

### ISSUE THREAD

**Issue by ellispritchard** [Financial-Times/n-makefile#111]
I've added a `.png` file to my repo, for an architecture diagram.

Unfortunately the `_verify_lintspaces` target tries to run `lintspaces` on every git recorded file, including the PNG file, which obviously breaks the build.

`lintspaces` should only run on text files, or restrict to known extensions, rather than on the entire output of `git ls-files -z $1 | tr '\0' '\n' | xargs -I {} find {} ! -type l`

**Comment by matthew-andrews** [comment]
hmm, i think the command to execute lintspaces is correct but instead we just need to tweak the `.editorconfig` file.  Are you using the default `.editorconfig` file or you own version?

Suspect it's this line in the `.editorconfig` file causing the pain:-
https://github.com/Financial-Times/n-makefile/blob/master/config/.editorconfig#L3

**Comment by ellispritchard** [comment]
The default from `make updatetools`

**Comment by matthew-andrews** [comment]
yup, let's tweak that - https://github.com/Financial-Times/n-makefile/blob/master/config/.editorconfig#L3
